src/platform/site-wide/announcements/components/HomepageRedesignModal.jsx

```diff
@@ -0,0 +1,74 @@
+import { VaButton } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+import Modal from '@department-of-veterans-affairs/component-library/Modal';
+
+import React from 'react';
+import { connect } from 'react-redux';
+import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
+import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
+import PropTypes from 'prop-types';
+import recordEvent from 'platform/monitoring/record-event';
+
+export function HomepageRedesignModal({ dismiss, vaHomePreviewModal }) {
```

Comment:
**MUST**
This does not need to be exported. More importantly, it _shouldn't_ be exported. The functionality (after being wrapped in a HOC) is being exported as the default at the bottom of the file. See comments on file where this functionality is being imported as well.

Reply:
removed export, thanks!